cloudformation/template.yaml

```diff
@@ -54,8 +54,14 @@ Parameters:
     Description: If you provided a domain name associated with an acm cert, then you must also specify here the acm cert's arn. Leave this blank to create a developer portal without a custom domain name.
     Default: ''
 
+  SkipRoute53NameserverCreation:
+    Type: Boolean
+    Description: Only applicable if creating a custom domain name for your dev portal. If true, skips creating a Route53 HostedZone and RecordSet. You'll need to provide your own nameserver hosting in place of Route53.
+    Default: false
```

Comment:
@a-tan Do we want this to be the default mode?

Reply:
I think the default mode should be AWS-managed & all-inclusive, which is why it defaults false. Curious about Alan's opinion!

Reply:
I think it'd be worth it to call this out as part of the deploy step in the README - `if you want to use a custom domain name and already own the domain blah blah blah....`

Reply:
Agreed! I'll add to the README